package io.novafoundation.nova.common.base.errors

class SigningCancelledException : Exception()
